src/components/scenes/GettingStartedScene.tsx

```diff
@@ -150,8 +149,8 @@ export const GettingStartedScene = (props: Props) => {
 
   // Initialize variant config values
   useAsyncEffect(async () => {
-    setIsFinalSwipeEnabled(Boolean(await getStickyRemoteConfigValue('swipeLastUsp')))
-    setCreateAccountType((await getStickyRemoteConfigValue('createAccountType')) as CreateAccountType)
+    setIsFinalSwipeEnabled(Boolean(await getStickyConfigValue('swipeLastUsp')))
+    setCreateAccountType((await getStickyConfigValue('createAccountType')) as CreateAccountType)
```

Comment:
If `getStickyConfigValue` had the this signature:

```ts
function getStickyConfigValue<K extends keyof StickyConfig>(key: K): Promise<StickyConfig[K]> 
```

Then there'd be no need for the type casting nor the type assertion.